mailbox/api/src/main/java/org/apache/james/mailbox/model/SearchQuery.java

```diff
@@ -284,31 +284,31 @@ public static Criterion modSeqEquals(long value) {
      * 
      * @param date
      *            given date
-     * @param res
+     * @param dateResolution
      *            the date resolution, either {@link DateResolution#Year},
      *            {@link DateResolution#Month}, {@link DateResolution#Day},
      *            {@link DateResolution#Hour}, {@link DateResolution#Minute} or
      *            {@link DateResolution#Second}
      * @return <code>Criterion</code>, not null
      */
-    public static Criterion internalDateAfter(Date date, DateResolution res) {
-        return new InternalDateCriterion(new DateOperator(DateComparator.AFTER, date, res));
+    public static Criterion internalDateAfter(Date date, DateResolution dateResolution) {
```

Comment:
I just discover this class. IMO the name is terrible I had to check at this code to see it just was a enum. What is it supposed to be a ranch for the search?